.eslintrc.js

```diff
@@ -12,7 +12,7 @@ const globals = {
 }
 
 module.exports = {
-    ignorePatterns: ['node_modules', 'plugin-server'],
+    ignorePatterns: ['node_modules', 'plugin-server', 'cypress'],
```

Comment:
Had to add this as ESLint was complaining about issues due to our Cypress code not being properly typed